v4l2grab.c

```diff
@@ -343,6 +347,19 @@ static int frameRead(void)
 	return 1;
 }
 
+/**
+	delay (ms)
+*/
+static void delay(int milliseconds) {
```

Comment:
Any reason for introducing a custom busy-wait instead of using `usleep` (https://man7.org/linux/man-pages/man3/usleep.3.html)?

Reply:
Not a good one. I'll update it in a little bit and create a new request. I'll just close this one for now. Thanks!